test/unit/iXmlUnit.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require('chai');

```

Comment:
Hi @jkdavew 

Since `iXmlNodeParmOpen` is only used in `iPgm.addParam()` I would move these tests into the existing [iPgmUnit](https://github.com/IBM/nodejs-itoolkit/blob/master/test/unit/iPgmUnit.js) rather than a separate file.